library/src/text/raw.rs

```diff
@@ -197,6 +198,18 @@ impl Finalize for RawElem {
     }
 }
 
+impl LocalName for RawElem {
+    fn local_name(&self, lang: Lang) -> &'static str {
+        match lang {
+            Lang::CHINESE => "代码",
+            Lang::GERMAN => "Computercode",
+            Lang::ITALIAN => "codice ",
```

Comment:
why the trailing space and lowercase?

Reply:
BTW, we would need someone that speaks Russian, German, Italian and Chinese to double check these, I use linguee so I am guessing they are mostly okay, but in Chinese and Russian I don't know how (if at all) capitalization works. And just to double check translations.